rclcpp/include/rclcpp/timer.hpp

```diff
@@ -108,6 +112,36 @@ class GenericTimer : public TimerBase
     cancel();
   }
 
+  void
+  execute_callback()
+  {
+    execute_callback_delegate<>();
+  }
+
+  // void specialization
+  template<typename CallbackT = FunctorT,
+  typename std::enable_if<
+    rclcpp::function_traits::same_arguments<CallbackT, VoidCallbackType>::value
+  >::type * = nullptr
```

Comment:
It does; in fact, I think uncrustify changed my indentation to this from something like

```
  template<typename CallbackT = FunctorT,
    typename std::enable_if<
      rclcpp::function_traits::same_arguments<CallbackT, VoidCallbackType>::value
    >::type * = nullptr
  >
```

which I find more readable.